src/apps/Altinn.AccessManagement/test/Bruno/AccessMgmt/Automatic Test Collection/Meta/roles/GET_roles_rolepackages.bru

```diff
@@ -0,0 +1,25 @@
+meta {
+  name: GET_roles_rolepackages
+  type: http
+  seq: 5
+}
+
+get {
+  url: {{baseUrl}}/accessmanagement/api/v1/meta/info/roles/:id/packages
+  body: none
+  auth: inherit
+}
+
+params:path {
+  id: f76b997a-9bd8-4f7b-899f-fcd85d35669f
+}
+
+tests {
+  test("GET_roles_rolepackages", function() {
+    const data = res.getBody();  
+    expect(res.status).to.equal(200);
+    expect(data[0]).to.have.property('id', '01961576-f9b3-731a-b537-03fdad2239de');
```

Comment:
Her burde du heller sjekke på forventet kobling mellom RoleId og PakkeId stemmer.
Og antall dersom du skal sjekke for løsning på buggen: #758 

Id her er vel primarykey blir opprettet på ingest tidspunkt, og er da også er forskjellig mellom AT22 og TT02

Reply:
Oppdaterte testen